R/plot_sankey.R

```diff
@@ -63,7 +63,7 @@ plot_sankey <- function(data,
     geom_stratum() +
     geom_text(stat = StatStratum, aes(label = after_stat(.data$stratum))) +
     fill_score_colors() +
-    theme_minimal() +
```

Comment:
Very nice work Linda -- moving the implementation details elsewhere and reusing the high-level idea of a "tilt theme".